config/checker/com.intellij.astub

```diff
@@ -129,8 +129,6 @@ interface Application {
 @UIPackage package com.intellij.openapi.editor.colors;
 @UIPackage package com.intellij.openapi.externalSystem;
 @UIPackage package com.intellij.openapi.fileChooser;
-@UIPackage package com.intellij.openapi.fileEditor;
```

Comment:
Whoops... see the docs for methods in `com.intellij.openapi.fileEditor.FileDocumentManager`, e.g. `#saveAllDocuments`... that's one of the few places in IntelliJ API where they actually documented the call-me-on-EDT requirement ;D

Reply:
Better mark the specific class you know doesn't require EDT (`NonProjectFileWritingAccessProvider`?) as `@AlwaysSafe`/method as `@SafeEffect` instead